xAPI.md

```diff
@@ -510,6 +513,63 @@ Clarifications and additional examples including:
 - Addiitional explanitory text and diagrams
 [1.0.2...1.0.3](https://github.com/adlnet/xAPI-Spec/compare/1.0.2...1.0.3)
 
+
+<a name="Appendix1B"/>
+### Appendix B: cmi5 Example 
+
+The following example from the cmi5 Community of Practice demonstrates Extensions, "category", etc.
+
+```
+{
+  "id":"2a41c918-b88b-4220-20a5-a4c32391a240",
+  "actor": {
+    "objectType": "Agent",
+    "account": {
+      "homePage": "http://example.adlnet.gov",
+      "name": "1625378"
+    }
+  },
+  "verb": {
+    "id": "http://w3id.org/xapi/verbs/failed",
+    "display": {
+      "en-US": "Failed"
+    }
+  },
+  "object": {
+    "id":"https://mydomain.adlnet.com/AUidentifier",
```

Comment:
As @brianjmiller commented on an outdated diff, you've used example.adlnet.gov on line 528, you should continue to use it.